README.md

```diff
@@ -0,0 +1,19 @@
+# Small python docker image
+
+Based on alpine.
+
+See https://github.com/gliderlabs/docker-alpine
+
+## Usage
+
+Just derive from the image in `FROM` and the current directory will be added to `/app/` in the image and pip packages will be installed.
```

Comment:
I'd prefer `added as /app/` instead of `added to /app/`.

The current way it's not clear whether mydir might become `/app/mydir`.